run.py

```diff
@@ -308,7 +313,7 @@ def predict(self, split='valid', mode='tail_batch'):
 
 			for step, batch in enumerate(train_iter):
 				sub, rel, obj, label	= self.read_batch(batch, split)
-				pred			= self.model.forward(sub, rel)
+				pred		    = self.model.forward(sub, rel)
```

Comment:
Please do not reformat code, as this will make it hard to review it. Could you remove all the formatting differences?